src/app.ts

```diff
@@ -16,15 +17,19 @@ async function start(): Promise<void> {
     let keyValueDataObject: IKeyValueDataObject;
 
     if (isNew) {
-        const fluidDocument = await Fluid.createContainer(containerId, [
-            KeyValueInstantiationFactory,
-        ]);
+        const fluidDocument = await Fluid.createDocument(
+            containerId,
```

Comment:
Feels a little odd to pass a container ID to createDocument...

Reply:
i think we changed the name to documentId in other examples